vhdbuilder/packer/install-dependencies.sh

```diff
@@ -41,11 +41,15 @@ systemctlEnableAndStart disk_queue || exit 1
 mkdir /opt/certs
 chmod 666 /opt/certs
 systemctlEnableAndStart update_certs.path || exit 1
```

Comment:
dumb question: does enabling the timer automatically enable the service? I think it is directly enable-able, right?

honestly not sure

Reply:
haven't been able to find definitive answers online, I'm not sure either, but from what I've observed the target service does indeed get invoked by the timer after the timer has been enabled and started